bin/raygun_cli.dart

```diff
@@ -1,8 +1,9 @@
 import 'package:args/args.dart';
 import 'package:raygun_cli/sourcemap/sourcemap_command.dart';
 import 'package:raygun_cli/symbols/flutter_symbols.dart';
+import 'package:raygun_cli/deployments/deployments_command.dart';
 
-const String version = '0.0.2';
+const String version = '0.0.3';
```

Comment:
No need to change the version here, as it should match the one in https://github.com/MindscapeHQ/raygun-cli/blob/e43ed29bdbd5de40d28f126bc1c989d5448616b5/pubspec.yaml#L4 but I will increase it when we do a new release